tla-io/src/main/scala/at/forsyte/apalache/io/annotations/AnnotationParserError.scala

```diff
@@ -2,6 +2,6 @@ package at.forsyte.apalache.io.annotations
 
 class AnnotationParserError(message: String) extends Exception(message) {
   override def toString: String = {
-    "Error parsing annotation: " + message
+    "Annotation error: " + message
```

Comment:
Would it still be useful to signal that this is an error at the syntax level, rather than an error with the meaning of the annotation?

```suggestion
    "Annotation syntax error: " + message
```